lib/rubocop/cop/performance/caller.rb

```diff
@@ -10,20 +10,24 @@ module Performance
       #   # bad
       #   caller[1]
       #   caller.first
+      #   caller_locations[1]
+      #   caller_locations.first
       #
       #   # good
       #   caller(2..2).first
       #   caller(1..1).first
+      #   caller_locations(2..2).first
+      #   caller_locations(1..1).first
       class Caller < Cop
-        MSG_BRACE =
-          'Use `caller(%<n>d..%<n>d).first` instead of `caller[%<m>d]`.'.freeze
-        MSG_FIRST =
-          'Use `caller(%<n>d..%<n>d).first` instead of `caller.first`.'.freeze
+        MSG_BRACE = 'Use `%<method>s(%<n>d..%<n>d).first`' \
+                    ' instead of `%<method>s[%<m>d]`.'.freeze
+        MSG_FIRST = 'Use `%<method>s(%<n>d..%<n>d).first`' \
+                    ' instead of `%<method>s.first`.'.freeze
 
         def_node_matcher :slow_caller?, <<-PATTERN
           {
-            (send nil :caller)
-            (send nil :caller int)
+            (send nil {:caller :caller_locations})
```

Comment:
Yeah. We'd need to come up with something that is available. Will look into this later. 🙂